src/prettytable/prettytable.py

```diff
@@ -1638,15 +1638,21 @@ def _set_random_style(self) -> None:
     # DATA INPUT METHODS         #
     ##############################
 
-    def add_rows(self, rows: Iterable[RowType]) -> None:
+    def add_rows(self, rows: Iterable[RowType], divider: bool = False) -> None:
```

Comment:
```suggestion
    def add_rows(self, rows: Iterable[RowType], *, divider: bool = False) -> None:
```

To avoid calls like this, where it's not obvious what `True` is:
```python
table.add_rows(
    [
        ["Adelaide", 1295, 1158259, 600.5],
        ["Brisbane", 5905, 1857594, 1146.4],
        ["Darwin", 112, 120900, 1714.7],
    ],
    True,
)
```

Reply:
Interesting, wasn't aware of that pattern. I'll add it.